rskj-core/src/main/java/co/rsk/rpc/exception/JsonRpcTimeoutError.java

```diff
@@ -0,0 +1,12 @@
+package co.rsk.rpc.exception;
+
+import static com.googlecode.jsonrpc4j.ErrorResolver.JsonError.INTERNAL_ERROR;
+
+public class JsonRpcTimeoutError extends Error {
+    public static final int ERROR_CODE = INTERNAL_ERROR.code;
```

Comment:
I guess it'd make sense to create a custom error code for this too, wdyt? same as for `JsonRpcResponseLimitError`